docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: '3.1'
+
+services:
+
+  postgres:
+    restart: always
+    image: postgres:11.4-alpine
+    expose:
+      - "5432"
+    environment:
+      - POSTGRES_USER=seven23
+      - POSTGRES_PASSWORD=password
+      - POSTGRES_DB=seven23
+
```

Comment:
```suggestion
      - ALLOW_ACCOUNT_CREATION=True

```

Reply:
Yes, I saw the ALLOW_ACCOUNT_CREATION option but in the server part on line 26 is the right place, isn't it?

Reply:
Yes for sure, my mistake.